homeassistant/components/intent/__init__.py

```diff
@@ -420,6 +421,17 @@ async def async_handle(self, intent_obj: intent.Intent) -> intent.IntentResponse
         return response
 
 
+class HelloIntentHandler(intent.IntentHandler):
+    """Responds with no action."""
+
+    intent_type = intent.INTENT_RESPOND
+    description = "Returns the provided response with no action."
+
+    async def async_handle(self, intent_obj: intent.Intent) -> intent.IntentResponse:
+        """Return the provided response, but take no action."""
+        return intent_obj.create_response()
```

Comment:
Tests are missing.

Reply:
Waiting on an intents bump before the test can be added.

Reply:
Tests added in https://github.com/home-assistant/core/pull/129830